dspace-server-webapp/src/main/java/org/dspace/app/rest/signposting/processor/item/ItemTypeProcessor.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.signposting.processor.item;
+
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.dspace.app.rest.signposting.model.LinksetNode;
+import org.dspace.app.rest.signposting.model.LinksetRelationType;
+import org.dspace.content.Item;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.util.FrontendUrlService;
+import org.dspace.util.SimpleMapConverter;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * An extension of {@link ItemSignpostingProcessor} for the type relation.
+ * Provides links to a specific type from schema.org.
+ */
+public class ItemTypeProcessor extends ItemSignpostingProcessor {
+
+    private static final Logger log = Logger.getLogger(ItemTypeProcessor.class);
+    private static final String ABOUT_PAGE_URI = "https://schema.org/AboutPage";
+
+    @Autowired
+    private SimpleMapConverter mapConverterDSpaceToSchemaOrgUri;
+
+    @Autowired
+    private ItemService itemService;
+
+    public ItemTypeProcessor(FrontendUrlService frontendUrlService) {
+        super(frontendUrlService);
+        setRelation(LinksetRelationType.TYPE);
+    }
+
+    @Override
+    public void addLinkSetNodes(Context context, HttpServletRequest request,
+                                Item item, List<LinksetNode> linksetNodes) {
+        try {
+            linksetNodes.add(new LinksetNode(ABOUT_PAGE_URI, getRelation(), "text/html", buildAnchor(context, item)));
```

Comment:
According to https://signposting.org/FAIR/ AboutPage should be added
> Provide a second link that has as target https://schema.org/AboutPage in the common case where an actual landing page is concerned. Do not provide this second link in case a content resource is concerned, i.e. do not provide this second link when the persistent identifier of the scholarly object directly resolves to a content resource, such as an HTML article, without presenting an intermediate landing page.